datatableview/views/base.py

```diff
@@ -98,12 +97,21 @@ class AutoMeta:
         kwargs = self.get_datatable_kwargs(**kwargs)
         for meta_opt in opts.__dict__:
             if meta_opt in kwargs:
-                setattr(opts, meta_opt, kwargs.pop(meta_opt))
+                if not (meta_opt == 'ordering' and kwargs['ordering'] is None):
+                    setattr(opts, meta_opt, kwargs.pop(meta_opt))
```

Comment:
this extra if keeps `ordering` from `meta` from being overridden by an invalid value from the view class. it was needed.